packages/node/src/plugins/segmentio/publisher.ts

```diff
@@ -97,6 +109,7 @@ export class Publisher {
   }
 
   flushAfterClose(pendingItemsCount: number) {
+    this._tokenManager?.stopPoller()
```

Comment:
Now that it's using an unref'd timer instead of a while loop, this isn't critical to shutdown.  We can leave it in for cleaning up after ourselves, but I wouldn't want to change too much code for that.

Reply:
> this isn't critical to shutdown
Then we should remove it IMO -- otherwise it will just create confusion.

Reply:
True - it's not really cleaning up after ourselves with how it is currently being called (except in the scenario there are 0 events waiting to be sent)